pyproject.toml

```diff
@@ -8,9 +8,10 @@ include = [ "nixops_gcp/nix/*.nix" ]
 
 [tool.poetry.dependencies]
 python = "^3.7"
-apache-libcloud = "^3.0.0"
+apache-libcloud = "^3.1.0"
 cryptography = "^2.9.0"
 nixops = {git = "https://github.com/NixOS/nixops.git"}
+#nixos-modules-contrib = {git = "https://github.com/nix-community/nixos-modules-contrib.git"}
```

Comment:
I guess this is wasn't meant to be added ?

Reply:
This is related to the 'autoLuks' as explained in the other [comment](https://github.com/nix-community/nixops-gce/pull/9#discussion_r442550577)

We need the [nixos-modules-contrib](https://github.com/nix-community/nixos-modules-contrib) 'plugin' to take advantage of the autoLuks feature used in GCP

This works locally but the problem is that the ci build would fail given that the plugin does not have its own _poetry.lock_

cc @adisbladis do you think once we add the missing poetry.lock, can we safely add this as a dependency for nixops-gcp?

Reply:
Done per https://github.com/nix-community/nixos-modules-contrib/commit/6e4d21f47f0c40023a56a9861886bde146476198